CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+# Contributing
+
```

Comment:
@kgashok, could you remove this file please?

Reply:
done!